### PR TITLE
Fix Dark mode

### PR DESCRIPTION
**Pull request by ptdatta**
**Notes for Reviewers**

Previous:
image

Now: 
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by theBeginner86 — CHANGES_REQUESTED** [review]
Please consider fixing the failing lint checks
https://github.com/meshery/meshery/actions/runs/6784812335/job/18441838536?pr=9318#step:3:32

File: ui/components/MeshModelRegistry/MeshModelComponent.js [hunk 1]
```diff
@@ -334,8 +334,8 @@ const MeshModelComponent = ({ modelsCount, componentsCount, relationshipsCount }
             elevation={3}
             className={`${StyleClass.cardStyle} ${animate ? StyleClass.cardStyleAnimate : ''}`}
             style={{
-              backgroundColor: `${view === MODELS && animate ? 'white' : ''}`,
-              color: `${view === MODELS && animate ? 'black' : ''}`,
+              backgroundColor: `${view !== MODELS && animate ? '#677a84' : ''}`,
+              color: `${view !== MODELS && 'white'}`,
```
Comment (theBeginner86) [inline ui/components/MeshModelRegistry/MeshModelComponent.js:338]: Can we please consider using our theme? 

You can define these new colors in our theme and refer them as variables.

Comment (ptdatta) [reply ui/components/MeshModelRegistry/MeshModelComponent.js:338]: Yes Sure.

**Comment by abhijeetgauravm** [comment]
Hey @ptdatta  Let's discuss this on Meshery Development Meeting tomorrow at 8:30 PM IST / 9 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#).

File: ui/components/MeshModelRegistry/MeshModelComponent.js (hunk 1 above)
Comment (ptdatta) [reply ui/components/MeshModelRegistry/MeshModelComponent.js]: @theBeginner86 I changes all the color codes to theme references.

**Review by theBeginner86 — APPROVED** [review]
LGTM

File: ui/components/MeshModelRegistry/MeshModelComponent.js (hunk 1 above)
Comment (theBeginner86) [reply ui/components/MeshModelRegistry/MeshModelComponent.js]: Thank you!

**Pull request merged**